### PR TITLE
fix for #9587

### PR DESCRIPTION
**Pull request by l0stb1t**

### CODE REVIEW

File: libr/anal/p/anal_x86_cs.c [hunk 1]
```diff
@@ -204,24 +204,8 @@ static char *getarg(struct Getarg* gop, int n, int set, char *setop, int sel) {
...
-#if 0
-				case X86_REG_ESP: op.reg = X86_REG_RSP; break;
-				case X86_REG_EBP: op.reg = X86_REG_RBP; break;
-#endif
-				default: break;
-				}
-			}
-			return (char *)cs_reg_name (handle, op.reg);
 		}
+		return (char *)cs_reg_name (handle, op.reg);
```
Comment (radare) [inline libr/anal/p/anal_x86_cs.c]: the pr makes sense, not sure why we had this hack around capstone, let's wait for travis and see

**Comment by XVilka** [comment]
Uh-oh, breaks 11 tests, though couple were fixed.

**Comment by radare** [comment]
not going to merge in this release . so this was a workaround for capstone

**Comment by XVilka** [comment]
Release was done, what do we do with this one? @l0stb1t any ideas?

**Comment by l0stb1t** [comment]
I will try to fix it and make another pull request.

**Comment by radare** [comment]
you can push in this pr

**Comment by radare** [comment]
hello?

**Comment by radare** [comment]
@l0stb1t ping

**Pull request closed without merging**